content_cafe.py

```diff
@@ -27,18 +30,23 @@
 class ContentCafeCoverageProvider(CoverageProvider):
     def __init__(self, _db):
         self._db = _db
-        self.input_source = DataSource.lookup(_db, DataSource.CONTENT_CAFE)
-        self.output_source = self.input_source
+        self.input_identifier_types = [Identifier.ISBN]
```

Comment:
I think we don't actually want to get Content Cafe coverage for every single ISBN the metadata wrangler knows about. We only want to get coverage for ISBNs that a client specifically asks about. However, I think this code is fine because we're not going to actually run this like a normal coverage provider, we're going to use ensure_coverage on individual ISBNs.